tiledb/sm/c_api/tiledb.cc

```diff
@@ -4126,6 +4151,330 @@ int32_t tiledb_subarray_get_range_var_from_name(
   return TILEDB_OK;
 }
 
+/* ****************************** */
+/*      SUBARRAY PARTITIONER      */
+/* ****************************** */
+
+int32_t tiledb_subarray_partitioner_alloc(
+    tiledb_ctx_t* ctx,
+    const tiledb_subarray_t* subarray,
+    tiledb_subarray_partitioner_t** subarray_partitioner,
+    // TBD: require these here, or provide a separate set_memory_budget
```

Comment:
Seems like you already went the "provide a separate set_memory_budget" way so this comment could be removed?

Reply:
removed